docs/uploading_and_publishing/explanation_how_the_processing_works.md

```diff
@@ -0,0 +1,122 @@
+
+In the previous section, [How-to upload data](howto_upload_data.md), we showed that when the data was dropped in the Uploads page, a processing of the raw data was triggered.
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="/assets/uploading_and_publishing/gwupload.gif" alt="GW upload gif" width="90%" title="Click to zoom in">
+    </label>
+</div>
+
+
+Once the data is added to the upload, NOMAD interprets the files and divides them into two categories: **mainfiles** and **auxiliary files**. In the same upload, there might be multiple mainfiles and auxiliary files organized in a folder tree structure.
```

Comment:
Actually this is not quite true, only mainfiles are determined. 
The first step in processing is to determine all the main files in the upload. This is done by filtering each file in the upload with the parser specifications defined in the parser plug ins configuration file. Here one can specify, among others, the pattern for the file name, the pattern in the file header or the type of the file.

Reply:
I think I will not explain anything about plugins and patterns right now. Simply that NOMAD recognizes (somehow) certain files as mainfiles. I also fixed the auxiliary files thingy.

Changed to:

_Once the data is added to the upload, NOMAD interprets the files and determines which of them is a **mainfile**. Any other files in the upload can be viewed as **auxiliary files**. In the same upload, there might be multiple mainfiles and auxiliary files organized in a folder tree structure._